tests/verifyJWT.test.ts

```diff
@@ -1,26 +1,34 @@
 import { verifyJWT } from "../src/lib/utils/auth/verifyJWT";
 import { OTHENT_PUBLIC_KEY } from "../src/lib/utils/auth/verifyJWT";
+import { getAuth0URL } from "../src/lib/utils/auth/auth0";
 import axios from "axios";
 import * as dotenv from "dotenv";
 dotenv.config();
 
 test("if verifyJWT() works", async () => {
-  if (!process.env.auth0ClientId || !process.env.auth0ClientSecret) {
-    console.log("Please specify a auth0ClientId/auth0ClientSecret in the env.");
+  // TODO: These checks should run before starting the server in index.ts too. Also, the shape of some of these (e.g. auth0ClientDomain) should be validated too.
```

Comment:
Re-shape the Express "App" so that these checks run before / when starting the server.